### PR TITLE
Authpage

### PR DESCRIPTION
**Pull request by Varu98**

### CODE REVIEW

File: src/frontend/CartPage/CartPage.jsx [hunk 1]
```diff
@@ -0,0 +1,39 @@
+import React from "react";
+
+const CartPage = () => {
+  return (
+    <div className="flex-row">
+      <my-cart class="cart-container">
+        <table>
```
Comment (manojasarsa) [inline src/frontend/CartPage/CartPage.jsx]: Nice use of `table` element

File: src/frontend/Home/AuthPage/AuthPage.jsx [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+          <h2 class="align-center">Login</h2>
+          <div class="login-content">
+            <label for="email">Email address</label>
+            <input type="text" placeholder="abc@abc.com" />
+            <label for="password">Password</label>
+            <input type="password" />
+          </div>
+          <div class="login-footer">
+            <label for="remember">
+              <input type="checkbox" name="" id="" />
```
Comment (manojasarsa) [inline src/frontend/Home/AuthPage/AuthPage.jsx:18]: You can give values for name and id, they're empty

**Review by manojasarsa — APPROVED** [review]
Rest, looks good

File: src/frontend/WishlistPage/WishlistPage.jsx [hunk 3]
```diff
@@ -0,0 +1,16 @@
+import React from "react";
+import ProductCard from "../ProductCard/ProductCard";
+import "./WishlistPage.css";
+const WishlistPage = () => {
+  return (
+    <div>
+      {" "}
+      <h2 class="wishlist-title align-center">Wishlist</h2>
+      <div class="wishlist-container flex-row">
+        <ProductCard />
```
Comment (talrejah02) [inline src/frontend/WishlistPage/WishlistPage.jsx]: Nice use of repeated properties as classes,I will do the same

**Review by talrejah02 — APPROVED** [review]
Clean code, you just reminded me that I could use classes for repeated properties.

**Pull request merged**